### PR TITLE
Fixing Sources

### PR DESCRIPTION
**Pull request by xOnlyFadi**

### CODE REVIEW

File: sources for users/sources-dev/NepNep/mangasee/mangasee.js [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+            const name = elem.ChapterName ? elem.ChapterName : '' // can be null)
+
+            chapters.push({
+                title: name,
+                url: `/read-online/${id}`,
+                volume,
+                number: chapNum,
+                language: 'English',
+                date_upload: '',
+                scanlator: ''
```
Comment (chubimauk) [inline sources for users/sources-dev/NepNep/mangasee/mangasee.js]: please make scanlator null to fix the duplicate chapters issue

**Review by chubimauk — CHANGES_REQUESTED** [review]
Please remove the /NepNep/ folder everywhere.

**Comment by xOnlyFadi** [comment]
when removing /NepNep/ folder do i replace the original mangasee with mine?

**Comment by chubimauk** [comment]
> when removing /NepNep/ folder do i replace the original mangasee with mine?

yeah that's the idea of version control. The older version will always be there in the previous commit if necessary to look at

**Comment by xOnlyFadi** [comment]
Fixed things can you merge?

**Pull request closed without merging**